ledger/benches/bonded_mapping.rs

```diff
@@ -22,12 +22,13 @@ use console::{
     program::{Identifier, Literal, Plaintext, ProgramID, Value},
 };
 use ledger_committee::{MAX_DELEGATORS, MIN_DELEGATOR_STAKE};
+use ledger_store::ConsensusStore;
+use synthesizer::VM;
```

Comment:
Should we add dependency ordering to the pre-commit hooks? Not here but perhaps as a follow up issue.

Reply:
I think the more style-enforcing pre-commit hooks the better!